hwilib/hwwclient.py

```diff
@@ -33,6 +33,11 @@ def sign_message(self, message, keypath):
         raise NotImplementedError('The HardwareWalletClient base class does not '
                                   'implement this method')
 
+    # Display address on device display
```

Comment:
This abstract method was missing so figured I'd add it

Reply:
Shouldn't this also have a `redeem_script` argument? The other wallets that support `displayaddress` should then throw an error if that argument is present.

Reply:
Agreed that this should also have `redeem_script` and every other `HWWClient` should as well, just throwing an error if they see it.